examples/chart/teleport-kube-agent/Chart.yaml

```
@@ -1,6 +1,6 @@
 name: teleport-kube-agent
 apiVersion: v2
-version: 0.0.3
+version: 0.0.5
```

Just a single version bump is fine here.

```suggestion
version: 0.0.4
```